### PR TITLE
Fix month names for bangla and bengali

### PR DESCRIPTION
**Pull request by govindkrjoshi**
Story: https://www.pivotaltracker.com/story/show/170671424

### CODE REVIEW

**Comment by kitallis** [comment]
Can we just make use of the `abbr_month_names` array instead of repeating it?

File: config/locales/api/bn_BD.yml [hunk 1]
```diff
@@ -12,6 +12,19 @@ bn-BD:
       invalid_password: 'আপনার পাসওয়ার্ড মিলছে না। আবার চেষ্টা করবেন?'
   month_names: [~, 'জানুয়ারী', 'ফেব্রুয়ারী', 'মার্চ', 'এপ্রিল', 'মে', 'জুন', 'জুলাই', 'অগাস্ট', 'সেপ্টেম্বর', 'অক্টোবর', 'নভেম্বর', 'ডিসেম্বর']
   abbr_month_names: [~, 'জানুয়ারী', 'ফেব্রুয়ারী', 'মার্চ', 'এপ্রিল', 'মে', 'জুন', 'জুলাই', 'অগাস্ট', 'সেপ্টেম্বর', 'অক্টোবর', 'নভেম্বর', 'ডিসেম্বর']
+  months:
```
Comment (kitallis) [inline config/locales/api/bn_BD.yml]: Can we just make use of the `abbr_month_names` array instead of repeating it?

**Comment by harimohanraj89** [comment]
@kitallis @govindkrjoshi if this works, let's ship as is with the other PRs going out today. I will slap some pre-emptive approval to unblock.

**Comment by kitallis** [comment]
> @kitallis @govindkrjoshi if this works, let's ship as is with the other PRs going out today. I will slap some pre-emptive approval to unblock.

Did one better and actually fixed the issue for real.

**Comment by harimohanraj89** [comment]
😎

**Pull request merged**